Sources/Amplitude/ObjC/ObjCAutocaptureOptions.swift

```diff
@@ -0,0 +1,117 @@
+import Foundation
+
+@objc(AMPAutocaptureOptions)
+public final class ObjCAutocaptureOptions: NSObject, OptionSet {
```

Comment:
I think the OptionSet conformance on the objc api makes it a bit unwieldy, and its purpose was so that we could have a single object that worked across languages.
Either:
Remove the existing autocaptureoptions, and make this the only class you use across the sdk (changing naming as appropriate)
Or:
Remove OptionSet conformance and refactor the api.

Reply:
I picked the second option: f68eb0c0d991c20595c11072680eb8889fe0afe4.